readme.md

```diff
@@ -46,14 +30,37 @@ try {
 } finally {
     Tracer.completeSpan();
 }
+```
 
+In addition to cross-service call tracing, the `Tracer` library supports intra-thread tracing, for example:
+
+```java
+// Record tracing information for expensive doSomeComputation() call:
```

Comment:
this seems like the same as the existing example.  perhaps just add a comment there?